conda/pprof/LICENSE

```diff
@@ -0,0 +1,13 @@
+BSD 3-clause license
+Copyright (c) 2015-2018, conda-forge
```

Comment:
OK - I like the move to pprof here....So we've moved to using conda forge gperftools and then downloading and building pprof? Shouldn't this then be the google license?